low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -107,8 +107,9 @@ def main(product):
             section=SECTION,
             key='secondary_rabbitmq_server'
             )
-        _check_rabbitmq_status()
-        _start_rabbitmq()
+
+    _start_rabbitmq()
```

Comment:
you are resetting the order. if reset done atlast (its just killing beam process & not resetting), then it wont be started.

Reply:
It went in positive flow. So I couldn't find rabbitmq start required. But, yes we should start rabbitmq.
Added start_rabbitmq() after reset_rabbitmq() method call.